aws/solutions/lambda-backed-cloudformation-custom-resources/get_vpc_main_route_table_id/Routetable.py

```diff
@@ -0,0 +1,106 @@
+from __future__ import print_function
+import json
+import boto3
+import urllib
+from botocore.vendored import requests
+
+
+SUCCESS = "SUCCESS"
+FAILED = "FAILED"
+
+def send(event, context, responseStatus, responseData, physicalResourceId=None, noEcho=False):
+    responseUrl = event['ResponseURL']
+
+    print(responseUrl)
+
+    responseBody = {}
+    responseBody['Status'] = responseStatus
+    responseBody['Reason'] = 'See the details in CloudWatch Log Stream: ' + context.log_stream_name
+    responseBody['PhysicalResourceId'] = physicalResourceId or context.log_stream_name
+    responseBody['StackId'] = event['StackId']
+    responseBody['RequestId'] = event['RequestId']
+    responseBody['LogicalResourceId'] = event['LogicalResourceId']
+    responseBody['NoEcho'] = noEcho
+    responseBody['Data'] = responseData
+
+    json_responseBody = json.dumps(responseBody)
+    print("Response body:\n" + json_responseBody)
```

Comment:
Is this needed?

Reply:
Yes. This is taken from the samples provided